openpmd_validator/createExamples_h5.py

```diff
@@ -157,12 +157,14 @@ def write_rho_cylindrical(meshes, mode0, mode1):
 
     # Add information on the r-z grid
     rho.attrs["gridSpacing"] = np.array([1.0, 1.0], dtype=np.float32)  # dr, dz
-    rho.attrs["gridGlobalOffset"] = np.array([0.0, 0.0], dtype=np.float32) 
+    rho.attrs["gridGlobalOffset"] = np.array([0.0, 0.0], dtype=np.float32)
     rho.attrs["position"] = np.array([0.0, 0.0], dtype=np.float32)
-    rho.attrs["gridUnitSI"] = np.float64(1.0)
+    rho.attrs["gridUnitSI"] = np.array([1.0]*2, dtype=np.float64)
```

Comment:
Wow, interesting syntax...

maybe we can write `[1.0, 1.0]` for clarity?
or define a variable named `dataDim = 2` for `2` so it's not a magic number?